pio/ppm/ppm.c

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * 8-channel Radio Control PPM example
+ * Output on GPIO3
+ *
+ * 0.5ms   5ms         0.5ms      1-2mc      0.5ms      1-2mc
```

Comment:
Are the `1-2mc` here supposed to be `1-2ms`?
Thanks for incorporating my other suggestion :slightly_smiling_face:

Reply:
Thanks for the review and for the suggestion.
Yes, that is the idea of PPM protocol for radio-controlled models. 
See here, for instance: https://sourceforge.net/p/arduinorclib/wiki/PPM%20Signal/

Reply:
I wasn't querying the timing, I was only querying the use of `mc` versus `ms` ?

Reply:
omg... thanks

Reply:
Ok, that was just a documentation glitch. Now everything is fixed.